packages/types/src/modeOptions.ts

```diff
@@ -2,7 +2,7 @@ export interface CommonModeOptions {
   invokedFromCli: boolean
   userNodePath?: string
   userNodeVersion?: string
-  configFile?: false | string | null
+  configFile?: string | null
```

Comment:
How come this can be `null`? I wonder if we can do `configFile?: string` here. Not your change specifically but always good to question and improve code quality.

Reply:
It can be null here since you can opt to not pass in any value for `configFile` and if none is passed, we will assume you are using the default and populate it with the detected file. So after ProjectLifeCycleManager runs, `configFile` will be a string.